setup.cfg

```
@@ -0,0 +1,86 @@
+[metadata]
+name = ashley
+version = 0.0.1
+description = A self-hosted message board for OpenEdx
```

I would suggest adding the "forum" keyword.

```suggestion
description = A self-hosted alternative discussion forum for OpenEdx
```
The same applies for the project's README and github project description